packages/package_info_plus/package_info_plus/pubspec.yaml

```diff
@@ -33,14 +33,14 @@ dependencies:
   meta: ^1.3.0
   path: ^1.8.0
   package_info_plus_platform_interface: ^2.0.1
-  win32: ">=2.7.0 <5.0.0"
+  win32: ^4.0.0
```

Comment:
FYI there is already a PR that bumps win32 and environment as well: https://github.com/fluttercommunity/plus_plugins/pull/1731

Looking forward for your PR to be ready :)